### ISSUE TITLE
How to find the logits of previous network? 

### ISSUE THREAD

**Issue by qaazii** [kshmelkov/incremental_detectors#12]
Hi, Dear Author @kshmelkov Kindly explain the logits than are used as in input to the distillation loss in 2nd training. Please only point out the code that is taking the logits of the previous network as an input to the extended network. 

Thank you very much.
